assets/src/edit-story/elements/image/display.js

```diff
@@ -40,10 +40,11 @@ const Element = styled.div`
 const Img = styled.img`
   position: absolute;
   ${imageWithScale}
+  ${elementWithFlip}
```

Comment:
One suggestion: since flip is pretty rare, we could move it up inside the `displayElement.js` and `outputElement.js`, as a special sub-container. E.g.:

```
return (
  <Wrapper>
     <WithFlip flip={flip}>
        <Display ..../>
     </WithFlip>
  </Wrapper>
)
```

That way, most of moveable operations will not collide with this sub-container and you will need fewer places to  move it around, and it will automatically work for all types of elements. And it will disentangle from Moveable and `useTransformHandler` stuff.

The `edit.js` is a bit harder. I'm not 100% it'd work the same. Seems like it should, but we'd have to try. But even if it doesn't, it might still be easier to manage only 1 or 2 components, instead of all?

Reply:
I suppose it might be a bit less relevant now when it was confirmed that only Media will have flipping, however, did add a flip wrapper within `displayElement`. Moveable behavior got quite unexpected using it with edit though, kept it as it was for that case, seems to be more straightforward for handling Moveable.